app/frontends/ogc-coverages/util/subset-parameter-parsing.ts

```diff
@@ -67,26 +68,11 @@ const dimensionConfig: DimensionConfig = {
   },
 };
 
-/**
- * Tag class for denoting errors during parsing
- *
- */
-export class ParameterParseError extends Error {}
-
-/**
- * Helper function for parameters that parses and validates boolean values. A null value  
- * defaults to false.
- * 
- * @param valueStr - the unparsed boolean as it appears in the input
- * @returns the parsed result
- * @throws ParameterParserError - if there are errors while parsing
- */
-export function parseBoolean(valueStr: string): boolean {
-  if (!valueStr) return false;
-  if (valueStr.toLowerCase() === 'true') return true;
-  if (valueStr.toLowerCase() === 'false') return false;
-  throw new ParameterParseError('must be \'false\' or \'true\'');
-}
+const arbitraryDimensionConfig = {
+  lowToHigh: false,
```

Comment:
This means we'll allow min values greater than max values - it would be nice to turn this validation on, but don't want to break for some special case dimension where it's valid.

Reply:
Based on feedback from harmony-service-providers channel, I changed this code to perform the validation that min must be less than or equal to the max.